community_build.sh

```diff
@@ -0,0 +1,146 @@
+# COMMUNITY_BUILD.SH notes: 
+#       (1) the main goal is to build several community-build projects to test the performance of scala 3 compiler 
+#       (2) the name of the community-build projects need to be added to project_list 
+#       (3) the output of building community-build projects can be found at output/output_project_name.log
+#       (4) output/result_summary.log contain two lists of project names; one for those that are built successfully, and one for those that are not
+#       (5) the error message found in output/output_project_name.log are stored in output/error_project_name.log
+#       (6) output/error_summary_count contain a table summarizing the # of error found in each output/output_project_name.log
+#       (7) output/error_summary_message.log conatin a summary of all the errors found accross building all the projects
+
+#include all the community projects that need to be built in the following project_list
+project_list_1=("intent")
+project_list=("intent"
+            "scalacheck"
+            "scalatest" 
+            "scalatestplus-scalacheck"
+            "scalatestplus-junit" 
+            "scalatestplus-testng" 
+            "scala-xml" 
+            "scalap"
+            "betterfiles" 
+            "ScalaPB" 
+            "minitest" 
+            "fastparse" 
+            "stdLib213" 
+            "shapeless" 
+            "xml-interpolator" 
+            "effpi" 
+            "sconfig" 
+            "zio" 
+            "munit" 
+            "scodec-bits"
+            "scodec" 
+            "scala-parser-combinators" 
+            "dotty-cps-async" 
+            "scalaz" 
+            "endpoints4s" 
+            "cats-effect-3" 
+            "scala-parallel-collections" 
+            "scala-collection-compat"
+            "scala-java8-compat" 
+            "verify"  
+            "discipline"  
+            "discipline-munit" 
+            "discipline-specs2" 
+            "simulacrum-scalafix"  
+            "cats" 
+            "cats-mtl" 
+            "coop" 
+            "Equal" 
+            "FingerTree"
+            "Log" 
+            "Model" 
+            "Numbers" 
+            "Serial" 
+            "AsyncFile" 
+            "Span" 
+            "scala-stm" 
+            "Lucre" 
+            "izumi-reflect" 
+            "perspective" 
+            "akka" 
+            "Monocle" 
+            "protoquill" 
+            "onnx-scala" 
+            "play-json"
+            "munit-cats-effect" 
+            "scalacheck-effect" 
+            "fs2"  
+            "libretto"  
+            "jackson-module-scala"  
+            "specs2" 
+            "spire" 
+            "http4s" 
+            "parboiled2"
+        )
+
+#set the current folder name
+current_dir=test_community_build
+
+#set output to be the name of the directory that project.log and result_summary.log will be stored
+output=output_no_change
```

Comment:
Any new files or dirs that we create, let's make sure to add them to the `.gitignore`.